mPower2/mPower2TestApp/Info.plist

```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>2.4</string>
```

Comment:
Was this (changing to hardcoded values) intentional?

Reply:
Yes, used the agvtool to set the marketing version